apisix/plugins/ai-proxy-multi.lua

```diff
@@ -30,6 +30,7 @@ local ipairs = ipairs
 local type = type
 
 local priority_balancer = require("apisix.balancer.priority")
+local endpoint_regex = "^(https?)://([^:/]+):?(%d*)/?.*$"
```

Comment:
Are you sure this is correct?

1. The `/` appears to be unescaped. If the Lua string `"xxx"` can avoid this issue, please disregard.
2. `%d` seems to cause backtracking and result in a failed match? Whereas `\d` matches correctly.

I'm using https://regex101.com to check it.

Reply:
I guess its parsed differently in the website vs the lua code.
1. is no issue. its parsed correctly as can be confirmed by tests.
2. It does match otherwise tests would be failing.